MODULE.bazel

```diff
@@ -120,8 +120,32 @@ llvm = use_extension("@toolchains_llvm//toolchain/extensions:llvm.bzl", "llvm")
 
 CXX_STANDARD = "c++23"
 
+# This is based off of the default flags in toolchains_llvm
+# https://github.com/bazel-contrib/toolchains_llvm/blob/69d39965/toolchain/cc_toolchain_config.bzl#L125-L137
+COMMON_COMPILE_FLAGS = [
+    "-U_FORTIFY_SOURCE",
+    "-fstack-protector",
+    "-fno-omit-frame-pointer",
```

Comment:
We don't set this in cmake btw.

Reply:
Good callout. Do you think we should remove it?

Reply:
Probably yeah to fully match the cmake build in the beginning. It's probably something like a 1% perf overhead 🤷